src/main/java/htsjdk/samtools/SamPairUtil.java

```diff
@@ -57,31 +59,41 @@ public static enum PairOrientation
 
     /**
      * Computes the pair orientation of the given SAMRecord.
-     * @param r
+     * @param record the record for which the pair orientation is requested
      * @return PairOrientation of the given SAMRecord.
      * @throws IllegalArgumentException If the record is not a paired read, or
-     * one or both reads are unmapped.
+     * one or both reads are unmapped, or is the two records do are not mapped to the
```

Comment:
typo here: "or is the two records do are not"

Reply:
thanks. fixed.